sdk/ai/azure-ai-projects/README.md

```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
+Evaluation in Azure AI Project client library is desinged to assess the performance of generative AI applications in cloud. Generative AI application generations are quantitatively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as evaluators. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
```

Comment:
"Generative AI application generations" - generative / generations ... does not sound good to me.... Maybe say "The output of Generative AI application is quantitively..." or something similar